node.go

```diff
@@ -66,13 +66,14 @@ func New(cfg Config) (*Node, error) {
 	return n, nil
 }
 
-func (n *Node) Run() error {
+func (n *Node) Run(ctx context.Context) error {
 	// Configure tracing
 	if n.config.tracing {
-		if err := n.setupTracing(); err != nil {
+		if err := n.setupTracing(); err != nil { //nolint:contextcheck
```

Comment:
Why are we skipping the `contextcheck` linter here?

Reply:
It complains otherwise. setupTracing starts a goroutine which can't be cancelled, so there's no reason to wire through a context.